src/building/bootstrapping.md

```diff
@@ -448,3 +448,46 @@ This is an incomplete reference for the outputs generated by bootstrap:
 | copy `rustdoc`                                         | `build/HOST/stage2/bin`                                         |
 
 `--stage=2` stops here.
+
+### Clarification of build command's stdout
+
+In this part, we will investigate the build command's stdout in an action
+(similar, but more detailed and complete documentation compare to topic above).
+When you execute `x.py build --dry-run` command, the build output will be something
+like the following:
+
+```text
+Building stage0 library artifacts (aarch64-unknown-linux-gnu -> aarch64-unknown-linux-gnu)
+Copying stage0 library from stage0 (aarch64-unknown-linux-gnu -> aarch64-unknown-linux-gnu / aarch64-unknown-linux-gnu)
+Building stage0 compiler artifacts (aarch64-unknown-linux-gnu -> aarch64-unknown-linux-gnu)
+Copying stage0 rustc from stage0 (aarch64-unknown-linux-gnu -> aarch64-unknown-linux-gnu / aarch64-unknown-linux-gnu)
+Assembling stage1 compiler (aarch64-unknown-linux-gnu)
+Building stage1 library artifacts (aarch64-unknown-linux-gnu -> aarch64-unknown-linux-gnu)
+Copying stage1 library from stage1 (aarch64-unknown-linux-gnu -> aarch64-unknown-linux-gnu / aarch64-unknown-linux-gnu)
+Building stage1 tool rust-analyzer-proc-macro-srv (aarch64-unknown-linux-gnu)
+Building rustdoc for stage1 (aarch64-unknown-linux-gnu)
```

Comment:
I think using the `x86_64-unknown-linux-gnu` target/host here is better, since the `aarch64-unknown-linux-gnu` target is not as widely used and not Tier 1.

Reply:
I agree, `x86_64` is more commonly used.